e2e/rbd.go

```diff
@@ -487,6 +516,13 @@ var _ = Describe("RBD", func() {
 						go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 							name := fmt.Sprintf("%s%d", f.UniqueName, n)
 							wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+							filePath := appClone.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
+							checkSumClone := ""
+							checkSumClone, wgErrs[n] = calculateSHA512sum(f, &a, filePath, &opt)
+							e2elog.Logf("checksum value for the clone is %s", checkSumClone)
```

Comment:
The logs of failed e2e tests contain `checksum value for the clone is` without the value for checkSumClone, that means something went wrong before. Make sure to log the error here too.

Reply:
@yati1998 log error here so that we can see if something failed or not.